classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student)
```

Comment:
- [ ] Nicely done adding a student to a classroom. To make this method complete, we need to also set the classroom for the student. Kindly set the classroom attribute for the added student.